dnf.spec

```diff
@@ -1,44 +1,83 @@
+# default dependencies
 %global hawkey_version 0.13.3
 %global librepo_version 1.9.0
 %global libcomps_version 0.1.8
 %global rpm_version 4.14.0
+
+# conflicts
+%global conflicts_dnf_plugins_core_version 2.1.3
+
+# override dependencies for rhel 7
 %if 0%{?rhel} == 7
-%global rpm_version 4.11.3-27
+    %global rpm_version 4.11.3-27
```

Comment:
on Centos, there is only 4.11.3-25 version available. We require our own patched version 4.11.3-25.el7.centos.1 at the moment for centos builds.

Reply:
fixed